sqlacodegen/codegen.py

```diff
@@ -558,6 +558,8 @@ def render_column(self, column, show_name):
         elif has_index:
             column.index = True
             kwarg.append('index')
+        if column.comment:
```

Comment:
Separate with one empty line to distinguish the block from the previous if...else.